packages/grafana-toolkit/src/cli/index.ts

```diff
@@ -1,7 +1,6 @@
 import chalk from 'chalk';
 import { program } from 'commander';
 
-import { changelogTask } from './tasks/changelog';
```

Comment:
We also need to remove all the code associated to the command, in this case the `changelogTask` here `'./tasks/changelog'` and if there are other files associated that are only used there, delete them too.

We also want to see if there are nodejs dependencies that are used only there and nowhere else and if so, remove them from the package.json of the project.